packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -209,9 +208,7 @@ export const TrackScreenDetailsTile = ({
   uid,
   isLineupLoading
 }: TrackScreenDetailsTileProps) => {
-  const { hasStreamAccess, hasDownloadAccess } = useGatedContentAccess(
-    track as Track
-  ) // track is of type Track | SearchTrack but we only care about some of their common fields, maybe worth refactoring later
+  const { hasStreamAccess } = useGatedContentAccess(track as Track) // track is of type Track | SearchTrack but we only care about some of their common fields, maybe worth refactoring later
```

Comment:
`useGatedContentAccess` could just be updated to use `Pick<Track, 'fields' | 'we' | 'care' | 'about'>` and then it won't matter what type you pass in as long as it has those fields in it?

Reply:
whoa